queue_services/account-mailer/src/account_mailer/config.py

```diff
@@ -166,10 +166,9 @@ class TestConfig(_Config):  # pylint: disable=too-few-public-methods
     DB_PORT = os.getenv("DATABASE_TEST_PORT", "5432")
     os.environ["CLOUD_STORAGE_EMULATOR_HOST"] = "http://localhost:4443"
     os.environ["PUBSUB_EMULATOR_HOST"] = "localhost:8085"
-
-    SQLALCHEMY_DATABASE_URI = os.getenv(
-        "DATABASE_TEST_URL",
-        default=f"postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}",  # noqa: E231
+    SQLALCHEMY_DATABASE_URI = (
```

Comment:
Thank you Andriy

Reply:
avoids this setting:
https://github.com/bcgov/bcregistry-sre/blob/0281e37b6ddec3c2e3a8e21e4c8fdbddacd5008f/.github/workflows/backend-ci.yaml#L111